.github/ISSUE_TEMPLATE/release_checklist.md

```diff
@@ -35,11 +35,14 @@ We will refer to the release that is going out as the *current* release.
 - [ ] Create a branch off of master (DO NOT PUSH YET) for the *current* milestone of format `releases/v2.3` (where 2.3 is the *current* milestone). After that, anything landing in master will be part of the next milestone.
 - [ ] On the Release branch, pin the AC version to the stable version ([example](https://github.com/mozilla-mobile/fenix/commit/e413da29f6a7a7d4a765817a9cd5687abbf27619)) with commit message "Issue #`<this releng issue>`: Pin to stable AC `<version>` for release v2.3" (replacing 2.3 with the version)
 - [ ] Go through the list of issues closed during this sprint and make sure they all have the correct milestone.
```

Comment:
I heard back on the process for this.  People have been going through the commits between the last release and the current one, and checking all issues mentioned there.  Probably good to mention this in here.

(also good to know that this is pretty labor intensive, and should definitely be scriptable)

Reply:
[These are the issues merged since 2.3](https://github.com/mozilla-mobile/fenix/issues/6302#issuecomment-550548514)

Reply:
Actually with some work, we could automate this entire process.

User input: current milestone & previous release number
Script: Check the git log.  Find where the last release branched off.  Find every `#d+` pattern since then.  Using the GH API, check which are issues and which are PRs (filter out the latter).  For the former, set each to the current milestone.